itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/txn/compactor/TestCrudCompactorOnTez.java

```diff
@@ -68,10 +59,16 @@
 import org.junit.Assert;
 import org.junit.Test;
 
-import static org.apache.hadoop.hive.ql.txn.compactor.TestCompactor.executeStatementOnDriver;
+import java.io.IOException;
```

Comment:
why is the import reorder?

Reply:
There were some unused imports and I used Intellij Idea;s 'organize imports' command which orders and groups the imports as well on top of removing the unused ones.

Reply:
Reverted the reorder, only removed the unused ones